### ISSUE TITLE
wishlist: add qemu image 

### ISSUE THREAD

**Issue by raisjnn** [toltec-dev/toolchain#4]
it would be really cool if we were able to run a qemu image with arm and emulate the rM execution environment. we could then run tests, take screenshots and do other things from a CI machine

**Comment by fenollp** [comment]
Are you aware of https://github.com/evidlo/remarkable_sim ?

**Comment by raisjnn** [comment]
yes, we helped build resim by making sure it sent the proper events and the correct coordinates: https://github.com/rmkit-dev/remarkable_sim  

rmkit supports resim (is the only project to support it, afaik) bc resim was created to address the problems with rmkit''s viewer.py.

but resim is not enough, i want testing that simulates the device through qemu. perhaps using resim is a good first step though for projects that can work with resim

**Comment by raisjnn** [comment]
also, I'm not sure this is not a toolchain issue, it is for CI testing of built apps, to make sure installation works and other common tasks

**Comment by matteodelabre** [comment]
Well, it's a bit in-between, since if I understand correctly we'll need to have a replica of the reMarkable system for this to work. In particular, we'll need to have the same set of available libs.
